### ISSUE TITLE
Order of "Recently opened" and "File (and symbol) results" should be reversed by default and/or configurable

### ISSUE THREAD

**Issue by tchollingsworth** [microsoft/vscode#169362]
Does this issue occur when all extensions are disabled?: Yes

- VS Code Version: 1.74.1
- OS Version: Windows_NT x64 10.0.19044

Steps to Reproduce:

1. <kbd>CTRL</kbd>+<kbd>P</kbd> or use the Search box at the top of the window if enabled
2. Search for a very common filename that you would find in lots of different repositories/workspaces but typically only once in any given one, such as `package.json`

Expected results:

The files from the current workspace are autocompleted first, for example <kbd> CTRL</kbd>+<kbd>P</kbd> `package.json` <kbd>ENTER</kbd> should open the package.json from the root of the current project.

Actual results:

The most recently opened files are autocompleted first even if they are from a completely different project, for example <kbd>CTRL</kbd>+<kbd>P</kbd> `package.json` <kbd>ENTER</kbd> opens a package.json from some other project I opened yesterday rather than the project I am working on right now. 

You can turn off `search.quickOpen.includeHistory` but it's a useful feature, I often want to copy and paste from or look and what I did in another project.  And it's probably worth making an option since some people do work in huge monorepos where it useful to surface the most recently used files at the top, but then again those people could just set `search.quickOpen.history.filterSortOrder` to `recency` too.

**Comment by TylerLeonhardt** [comment]
I think this is basically a duplicate of https://github.com/microsoft/vscode/issues/5806 and is covered in https://github.com/microsoft/vscode/issues/27317

but I'll leave this open as a feature request.
